package.json

```diff
@@ -28,6 +28,8 @@
   },
   "dependencies": {
     "grunt": "~0.4.1",
+    "lodash": "^2.4.1",
+    "underscore.string": "^3.0.3",
```

Comment:
the problem i am having running tests is that unless i include underscore.string or lodash and cast it to the 
"_" variable then we end up with an undefined value in the code. However, there is no reason to have both in there. I am moving too fast trying to get this PR right. My apologies for that.